### ISSUE TITLE
Simple search

### ISSUE THREAD

**Issue by jmadin** [open-traits-network/open-traits-network.github.io#154]
Hi @jhpoelen and @BErfanian . I added a simple search here: http://opentraits.org/search.html
It's a hidden page, so we can do some testing / tweaking.

_Originally posted by @jmadin in https://github.com/open-traits-network/open-traits-network.github.io/issues/153#issuecomment-666186489_

**Comment by jhpoelen** [comment]
@jmadin your search prototype looks pretty neat! I imagine many other improvements (e.g., resolve taxon hierarchy of search term to include more specific taxa (e.g., when searching for Mammalia, also include datasets that only mention Homo).

But before trying to improve something that is already useful, why not integrate it in the dataset/member pages and solicit feedback via a link to a bug template? 

![Screenshot from 2020-08-17 12-41-21](https://user-images.githubusercontent.com/1084872/90437308-fc382980-e086-11ea-88f7-64b3ffa50228.png)

**Comment by jhpoelen** [comment]
@jmadin just checking in - still interested to add your search to the dataset pages? I think it'd be a neat addition.

**Comment by jmadin** [comment]
Oh yes, @jhpoelen. Just a matter of linking to the search page from a menu. The menu is getting crowded though. I'll take a look soon -- or feel free to do it yourself as I see you're doing some work on the code at the moment :).

**Comment by jhpoelen** [comment]
@jmadin I agree that the menu is getting busy. And, I think your search would be pretty useful in the existing page at https://opentraits.org/datasets . This way, there's no need to add some addition page. Is this what you had in mind?

**Comment by jmadin** [comment]
@jhpoelen That's a good idea. However, we'd need to make it operate on the existing table.

**Comment by jhpoelen** [comment]
You can use some javascript / css selectors to show/hide existing rows - it'll be super-fast.

**Comment by jhpoelen** [comment]
perhaps using a ```<div id="[datasetid]" ... class="visible"> [row content] </div>``` structure

**Comment by jhpoelen** [comment]
with  https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector .

**Comment by jhpoelen** [comment]
and dynamically add/removing the ```class="visible"``` attributes for rows of interest.
